src/Components/Confirmation/ConfirmationSteps.tsx

```diff
@@ -0,0 +1,589 @@
+        />
+      }
+      editAriaLabel={editExpensesAriaLabel}
+      stepName="hasExpenses"
+    >
+      {allExpenses()}
+    </ConfirmationBlock>
+  );
+}
+
+function Assets() {
+  const { formData } = useContext(Context);
+  const { formatMessage } = useIntl();
+  const translateNumber = useTranslateNumber();
+
+  const editAssetsAriaLabel = {
+    id: 'confirmation.assets.edit-AL',
+    defaultMessage: 'edit assets',
+  };
+  const assetsIconAlt = {
+    id: 'confirmation.expense.icon-AL',
+    defaultMessage: 'assets',
+  };
+
+  return (
+    <ConfirmationBlock
+      icon={<Resources title={formatMessage(assetsIconAlt)} />}
+      title={
+        <FormattedMessage
+          id="confirmation.displayAllFormData-householdResourcesText"
+          defaultMessage="Household resources"
+        />
+      }
+      editAriaLabel={editAssetsAriaLabel}
+      stepName="householdAssets"
+    >
+      <ConfirmationItem
+        label={
+          <FormattedMessage
+            id="confirmation.displayAllFormData-householdResourcesText"
+            defaultMessage="Household resources"
+          />
+        }
+        value={
+          <>
+            {translateNumber(formatToUSD(formData.householdAssets, 0))}
+            <i>
+              <FormattedMessage
+                id="confirmation.displayAllFormData-householdResourcesDescription"
+                defaultMessage="(This is cash on hand, checking or saving accounts, stocks, bonds or mutual funds.)"
+              />
+            </i>
+          </>
+        }
+      />
+    </ConfirmationBlock>
+  );
+}
+
+type BenefitList = {
+  [key: string]: {
+    name: FormattedMessageType;
+    description: FormattedMessageType;
+  };
+};
+
+type CategoryBenefits = {
+  [key: string]: { benefits: BenefitList; category_name: FormattedMessageType };
+};
+
+function HasBenefits() {
+  const { formData } = useContext(Context);
+  const { formatMessage } = useIntl();
+  const categoryBenefits = useConfig<CategoryBenefits>('category_benefits');
+
+  const alreadyHasBenefits = () => {
+    let allBenefits: BenefitList = {};
+
+    for (const category of Object.values(categoryBenefits)) {
+      allBenefits = { ...allBenefits, ...category.benefits };
+    }
+
+    const benefitsText = Object.entries(allBenefits)
+      .filter(([name, _]) => {
+        return formData.benefits[name as keyof BenefitsType];
+      })
+      .map(([name, value]) => {
+        return <ConfirmationItem label={value.name} value={value.description} key={name} />;
+      });
+
+    if (benefitsText.length > 0) {
+      return benefitsText;
+    }
+
+    return <FormattedMessage id="confirmation.none" defaultMessage="None" />;
+  };
+
+  const editHasBenefitsAriaLabel = {
+    id: 'confirmation.assets.edit-AL',
+    defaultMessage: 'edit benefits you already have',
+  };
+  const hasBenefitsIconAlt = {
+    id: 'confirmation.expense.icon-AL',
+    defaultMessage: 'benefits you already have',
+  };
```

Comment:
The aria label and icon alt are using the assets and expense ids. Is this intentional?